guides/bidirectionality.md

```diff
@@ -0,0 +1,22 @@
+# Angular Material bi-directionality
+
+### How to set your application direction
+The simplest way is applying `dir` attribute on either the `html` or `body` tags.
```

Comment:
@jelbourn 
> It will not emit for changes to `dir` on `<html>` and `<body>`, as these are assumed to be static.

 I'm not sure that this is a correct assumption to make, since overlayed items(`div.cdk-overlay-container`) are generated outside the Angular application context, so we don't have a single point to change the layout of the application. Or may be I have something missed?